charts/netdata/values.yaml

```diff
@@ -330,3 +330,130 @@ child:
     token: ""
     rooms: ""
     url: "https://app.netdata.cloud"
+
+k8sState:
+  enabled: false
+  port: "{{ .Values.parent.port }}"
+
+  resources: {}
+    # limits:
+    #  cpu: 4
+    #  memory: 4096Mi
+    # requests:
+    #  cpu: 4
+    #  memory: 4096Mi
+
+  livenessProbe:
+    failureThreshold: 3
+    periodSeconds: 30
+    successThreshold: 1
+    timeoutSeconds: 1
+  readinessProbe:
+    failureThreshold: 3
+    periodSeconds: 30
+    successThreshold: 1
+    timeoutSeconds: 1
+
+  terminationGracePeriodSeconds: 30
+
+  nodeSelector: {}
+
+  tolerations:
+    - operator: Exists
+      effect: NoSchedule
+
+  affinity: {}
+
+  priorityClassName: ""
+
+  podLabels: {}
+
+  podAnnotationAppArmor:
+    enabled: true
+
+  podAnnotations: {}
+
+  dnsPolicy: ClusterFirstWithHostNet
+
+  varLib:
```

Comment:
nit: I would omit `varLib`, because it is too enigmatic to end-user
```
  persistence:
    enabled: true
    ## Set '-' as the storageclass to get a volume from the default storage class.
    storageclass: "-"
    size: 1Gi
```